py/selenium/webdriver/common/__init__.py

```diff
@@ -14,3 +14,20 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+from .keys import Keys
```

Comment:
Well, actually, I figured out this import is useless as there is no instance of Keys that could be passed to the function as the Keys is just a holder of the values. So I will delete it. Therefore this code is unrelated to the Keys class and if you oppose the package file, I propose putting it into the utils.py file, since it is a utility function breaking things into lists.